src/develop/tutorial/ever-wallet.md

```diff
@@ -0,0 +1,220 @@
+---
+sidebar_position: 4
+---
+
+# Ever Wallet Guide
```

Comment:
This page should document [this](https://github.com/EverscaleGuild/everscale-tutor-web) project https://github.com/everscale-org/docs/issues/17